### ISSUE TITLE
Reproducing results

### ISSUE THREAD

**Issue by ahmadhassandebugs** [godka/comyco-lin#1]
Hi! I was trying to reproduce your results, but it didn't surpass the performance of Pensieve for all percentiles, as shown in the figure below. I trained the model for 500 epochs and it started to overfit after 480-490 epochs. I assume that is why you have set the total epochs to 500. The mean for comyco and pensieve is 0.936 and 0.925, respectively. Any idea how I can improve the performance further so that it is close to your pre-trained model performance. Thanks!

![cdf](https://github.com/godka/comyco-lin/assets/48770213/dcf6c35c-a496-47b3-bb53-0ccde22df110)

**Comment by godka** [comment]
Hi Ahmad,

https://github.com/godka/comyco-lin/blob/8cc945fb301a51f9c0cd7125fa648d2b3faef7c3/train.py#L115

Regarding the performances and overcoming the overfitting issues, the simplest way is to adjust the batch size via

```
actor_pool.get(2048) # batch_size = 2048, default: 256
```

Best,
Tianchi

**Comment by ahmadhassandebugs** [comment]
Sorry, I just saw your comment. I somewhat agree with you: policy gradient methods need larger batch sizes than typical deep supervised learning approaches in part due to the high variance. Increasing the batch size can reduce the variance of policy gradients. I will run comyco with different batch sizes and report the results here when I get time. Thanks!

**Comment by ahmadhassandebugs** [comment]
Here's what we get after training the model with a bigger batch size. It matches the results in paper I assume.

![cdf_batch2048](https://github.com/user-attachments/assets/fb0648c1-a52c-4912-81dd-5850ea17276c)
